src/benchmark.cc

```diff
@@ -562,6 +625,9 @@ void PrintUsageAndExit() {
           "          [--benchmark_out_format=<json|console|csv>]\n"
           "          [--benchmark_color={auto|true|false}]\n"
           "          [--benchmark_counters_tabular={true|false}]\n"
+          "          [--benchmark_console_id={true|false}]\n"
```

Comment:
please split this out to a separate PR. I'm not sure we will need it, and i'd like to keep this PR focused on the important bits.